pom.xml

```diff
@@ -226,6 +226,43 @@
       </build>
     </profile>
 
+    <profile>
+      <id>integration-test-vpc</id>
+      <build>
+        <plugins>
+          <plugin>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <configuration>
+              <skipTests>true</skipTests>
+            </configuration>
+          </plugin>
+          <plugin>
+            <artifactId>maven-failsafe-plugin</artifactId>
+            <executions>
+              <execution>
+                <goals>
+                  <goal>integration-test</goal>
+                  <goal>verify</goal>
+                </goals>
+              </execution>
+            </executions>
+            <configuration>
+              <includes>
+                <include>**/*IntegrationTest.java</include>
+              </includes>
+              <excludes>
+                <exclude>**/*GrpcIntegrationTest.java</exclude>
```

Comment:
Can we just exclude these tests in Maven CLI instead, because this is a temporary change?

Something like this:
```shell
mvn ... -Dit.test="!**/*GrpcIntegrationTest.java"
```